.identity/00_variables.tf

```diff
@@ -0,0 +1,53 @@
+variable "tags" {
+  type = map(any)
+}
+
+variable "prefix" {
+  type = string
+}
+
+variable "env" {
+  type = string
+}
+
+variable "env_short" {
+  type = string
+}
+
+variable "domain" {
+  type = string
+}
+
+variable "ci_github_federations" {
+  type = list(object({
+    repository        = string
+    credentials_scope = optional(string, "environment")
+    subject           = string
+  }))
+  description = "GitHub Organization, repository name and scope permissions"
+}
+
+variable "environment_ci_roles" {
+  type = object({
+    subscription    = list(string)
+    resource_groups = map(list(string))
+  })
+  description = "GitHub Continous Delivery roles"
```

Comment:
`Integration`

Reply:
fixed in [42a1bd0](https://github.com/pagopa/io-web-profile-backend/pull/32/commits/42a1bd07a2f1b3618634d703ecbf89c979516510)